File: ios/FileSystem.m
//
//  FileSystem.m
//  Real_Pitch
//
//  Created by iKooB on 9/2/16.
//  Copyright © 2016 Facebook. All rights reserved.
//

#import "RCTBridgeModule.h"
#import <Foundation/Foundation.h>

@interface RCT_EXTERN_MODULE(FileSystem, NSObject);

RCT_EXTERN_METHOD(getFoldersWithMusic:(RCTPromiseResolveBlock) resolve reject:(RCTPromiseRejectBlock) reject);

@end

File: ios/Real_Pitch/FileSystem.swift
//
//  FileSystem.swift
//  Real_Pitch
//
//  Created by iKooB on 9/2/16.
//  Copyright © 2016 Facebook. All rights reserved.
//

import Foundation

@objc(FileSystem)
class FileSystem : NSObject {
  @objc func getFoldersWithMusic(resolve: RCTPromiseResolveBlock, reject: RCTPromiseRejectBlock) -> Void {
    let fileManager = NSFileManager.defaultManager();
    let fileSystemEnum = fileManager.enumeratorAtPath("/Users");
    
    var nextObject = fileSystemEnum!.nextObject()!;
    
    while true {
      print(nextObject);
      nextObject = fileSystemEnum!.nextObject()!
    }
  }
}